src/c_api/c_api.cc

```diff
@@ -341,6 +412,8 @@ int MXLoadLib(const char *path) {
                             << "' custom op, Forward or CreateOpState function was not set.";
       CHECK(type_fp != nullptr) << "Error loading '" << name
                             << "' custom op, InferType function was not set.";
+      CHECK(stype_fp != nullptr) << "Error loading '" << name
+                            << "' custom op, InferSType function was not set.";
```

Comment:
Does it need to be set? or should we have a default to make all outputs dense and then let users specify it only when they are doing sparse?

Reply:
Based on my understanding, infer storage type is as important as infer data type.